src/Discord.Net.Interactions/Results/AutocompletionResult.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Discord.Interactions
+{
+    /// <summary>
+    ///     Contains the information of a Autocomplete Interaction result.
+    /// </summary>
+    public struct AutocompletionResult : IResult
+    {
+        /// <inheritdoc/>
+        public InteractionCommandError? Error { get; }
+
+        /// <inheritdoc/>
+        public string ErrorReason { get; }
+
+        /// <inheritdoc/>
+        public bool IsSuccess => Error is null;
+
+        /// <summary>
+        ///     Collection of Autocomplete suggestions to be displayed to the user
```

Comment:
Add missing `.` to end of summaries (applies to whole file)

Should be prefixed with `Gets`